index.html

```diff
@@ -2030,6 +2030,87 @@ <h4>Presentations Using Derived Credentials</h4>
             </figcaption>
           </figure>
         </section>
+        <section>
+          <h4>Presentations Including Holder Claims</h4>
+          <p>
+A <a>holder</a> MAY use the <code>verifiableCredential</code> <a>property</a> in
+a <a>verifiable presentation</a> to include <a>verifiable credentials</a> from
+any <a>issuer</a>, including themselves. When the <a>issuer</a> of a
+<a>verifiable credential</a> is the <a>holder</a>, its <a>claims</a> are
+considered to be self-asserted. These self-asserted claims can be secured by
+their own mechanism like any other <a>verifiable credential</a> or by the same
+mechanism that secures the <a>verifiable presentation</a> in which they are
+included.
+          </p>
+          <p>
+The <a>subject</a> of these self-asserted <a>claims</a> is not limited and can
+include statements about the <a>holder</a>, one of the other included
+<a>verifiable credentials</a>, or even the <a>verifiable presentation</a> in
+which the self-asserted <a>verifiable credential</a> is included. In each of
+these cases the <code>CredentialSubject</code> would have an <code>id</code>
+<a>property</a> that matches the <code>id</code> of the <a>subject</a>. This is
+no different than how it is done in <a>verifiable credentials</a> that are not
+self-asserted.
+          </p>
+          <p>
+A <a>verifiable presentation</a> that includes a self-asserted
+<a>verifiable credential</a> that is only secured using the same mechanism as
+the <a>verifiable presentation</a> MUST include a <code>holder</code>
+<a>property</a>.
+          </p>
+          <p>
+A self-asserted <a>verifiable credential</a> that has been included in the value
+of the <code>verifiableCredential</code> <a>property</a> of a
+<a>verifiable presentation</a> MUST include the following properties:
+          </p>
+
+        <dl>
+          <dt><var>type</var></dt>
+          <dd>
+The <code>type</code> <a>property</a> is required. One value MUST be
+<code>VerifiableCredential</code>, and it is RECOMMENDED that another
+value be <code>SelfAssertedCredential</code>. Additional types MAY also
+be included. For example,<br>
+<code>"type": ["VerifiableCredential", "SelfAssertedCredential", "ExampleTypeCredential"]</code>
+          </dd>
+          <dt><var>issuer</var></dt>
+          <dd>
+The <code>issuer</code> <a>property</a> is required. For self-asserted
+<a>verifiable credentials</a> that are secured using the same mechanism as the
+<a>verifiable presentation</a>, the value MUST be identical to the
+<code>holder</code> <a>property</a> of the <a>verifiable presentation</a>.
+Self-asserted <a>verifiable credentials</a> that use their own securing
+mechanism MUST follow the normative guidance for the <code>issuer</code>
+<a>property</a> in Section <a href="#issuer"></a>.
```

Comment:
This `MUST` + these `required`s seem redundant, especially when considering VCs in general (i.e., not self-asserted), which already require `type` and `issuer`.... If I'm not confused about this (please confirm!), I'll draft a rephrasing of lines 2062-2084.

Reply:
I would much appreciate a suggested rephrasing. The clarification was added in response to previous comments, and I am open to other possible ways of wording this.

Reply:
@TallTed in a fit of productivity, I have rephrased and attempted to eliminate the redundancies in this section. [bef11ba](https://github.com/w3c/vc-data-model/pull/1186/commits/bef11bab1f6d8dc111f10d669b529b99a50451ea)